src/vivarium_public_health/disease/special_disease.py

```diff
@@ -0,0 +1,134 @@
+    provided threshold. 
+    
+    In our provided examples, a person would no longer be experiencing 
+    ``protein_energy_malnutrition`` if their exposure drift out (or
+    changes via an intervention) of the provided exposure categories. 
+    Having your ``fasting_plasma_glucose`` drop below a provided level
+    does not necessarily mean you're no longer diabetic however.
+
+    Configuration defaults should be given as, for the continuous risk factor,
+
+    diabetes_mellitus:
+        threshold : 7
+        mortality : True
+        recoverable : False
+
+    For the categorical risk factor,
+
+    protein_energy_malnutrition:
+        threshold : ['cat1', 'cat2'] # provide the categories to get PEM.
+        mortality : True
+        recoverable : True
+    """
+
+    configuration_defaults = {
+        'risk_attributable_disease': {
+            'threshold': None,
+            'mortality': True,
+            'recoverable': True
+        }
+    }
+
+    def __init__(self, name, risk):
+        self.name = name
+        self.risk = risk
+        self.configuration_defaults = {
+            self.name: RiskAttributableDisease.configuration_defaults['risk_attributable_disease']
+        }
+
+    def setup(self, builder):
+        self.threshold = builder.configuration[self.name].threshold
+        self.recoverable = builder.configuration[self.name].recoverable
+        disability_weight = builder.data.load(f'cause.{self.name}.disability_weight')
+        self.distribution = builder.data.load(f'risk_factor.{self.risk}.distribution')
+
+        if builder.configuration[self.name].mortality:
+            csmr_data = builder.data.load(f'cause.{self.name}.cause_specific_mortality')
+            builder.value.register_value_modifier('csmr_data', lambda: csmr_data)
+            excess_mortality_data = builder.data.load(f'cause.{self.name}.excess_mortality')
+            builder.value.register_value_modifier('mortality_rate', self.mortality_rates)
+            self._mortality = builder.value.register_value_producer(
+                f'{self.name}.excess_mortality', source=builder.lookup.build_table(excess_mortality_data)
+            )
+
+        self._disability_weight = builder.lookup.build_table(disability_weight)
+        self.disability_weight = builder.value.register_value_producer(f'{self.name}.disability_weight',
+                                                                       source=self.compute_disability_weight)
+        builder.value.register_value_modifier('disability_weight', modifier=self.disability_weight)
+
+        self.exposure = builder.value.get_value(f'{self.risk}.exposure')
+        self.population_view = builder.population.get_view([self.name, 'alive'])
+
+        builder.event.register_listener('time_step', self.on_time_step)
+        builder.population.initializes_simulants(self.on_initialize_simulants)
+
+    def _get_population(self, index, query=None):
```

Comment:
Why are you duplicating the semantics of the population view?

Reply:
ah, since I didn't want to go through setup and wanted to find a way to test it and mock population view. I don't know if there's any other way to get around.

Reply:
Can't you just do:
```
disease = RiskAttributableDisease(...)
disease.population_view = mock_population_view
```
?

Reply:
I tried, but it says population_view is not a attribute of RiskAttributableDisease. I can try again, since I anyways need to update the tests after the changes.

Reply:
Should work.  This raises no errors:
```
>>> class A:
...     pass
... 
>>> a = A()
>>> a.thing = 'thing'
```